### PR TITLE
Fix/updates

### PR DESCRIPTION
**Pull request by dbjowett**

### CODE REVIEW

**Review by IslomJonMirzakbarov — APPROVED** [review]
cool eh

File: apps/web/src/pages/Profile/Profile.module.scss [hunk 1]
```diff
@@ -28,12 +28,10 @@
       .DescriptionTitle {
         font-weight: 600;
         font-size: 16px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Profile/Profile.module.scss:30]: How it comes to use template fonts?

File: apps/web/src/pages/SingleItem/SingleItem.module.scss [hunk 2]
```diff
@@ -201,102 +51,67 @@
           align-items: center;
 
           & .ItemTitle {
-            color: var(--sea-blue);
-            font-size: 12px;
+            color: var(--grey8);
+            font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/SingleItem.module.scss:55]: font template

File: apps/web/src/pages/SingleItem/SingleItem.module.scss [hunk 3]
```diff
@@ -201,102 +51,67 @@
...
             padding-bottom: 3px;
+            font-weight: 400;
           }
 
           & .ItemNumber {
-            color: var(--black);
+            color: var(--grey10);
             font-size: 22px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/SingleItem.module.scss:62]: font template

File: apps/web/src/pages/SingleItem/SingleItem.module.scss [hunk 4]
```diff
@@ -201,102 +51,67 @@
...
-        margin-top: 10px;
-        margin-bottom: 10px;
-      }
-      & .TagBtnContainer {
-        display: flex;
-        justify-content: space-between;
+        .Price {
+          width: 100%;
+          color: var(--main-blue);
+          font-size: 28px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/SingleItem.module.scss:103]: font template

File: apps/web/src/pages/SingleItem/SingleItem.module.scss [hunk 5]
```diff
@@ -201,102 +51,67 @@
...
           border: none;
-          border-radius: 40px;
-          cursor: pointer;
-          font-size: 12px;
-          height: 40px;
-          width: 30%;
+          border-radius: 5px;
+          background-color: var(--main-blue);
+          box-shadow: none;
+          font-size: 16px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/SingleItem.module.scss:114]: font template

File: apps/web/src/pages/SingleItem/Table/Table.module.scss [hunk 6]
```diff
@@ -44,7 +44,7 @@
       & .DateCell {
         color: var(--grey8);
         text-transform: uppercase;
-        font-size: 12px;
+        font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/SingleItem/Table/Table.module.scss:47]: font template

File: apps/web/src/pages/Wallet/TransComponents/Receive/index.tsx [hunk 7]
```diff
@@ -14,15 +13,6 @@ const seedData = {
 
 function copyText() {
   navigator.clipboard.writeText(seedData.walletAddress);
-  toast.info('Text copied to clipboard', {
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/TransComponents/Receive/index.tsx:17]: Why don't we need toast?

File: apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss [hunk 8]
```diff
@@ -62,41 +64,74 @@
...
-        padding-left: 25px;
-        letter-spacing: 0.005em;
-        color: var(--grey12);
+        padding-left: 35px;
+        color: var(--main-blue);
 
         font-weight: 600;
-        font-size: 16px;
-        line-height: 104.9%;
+        font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss:71]: font template

File: apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss [hunk 9]
```diff
@@ -62,41 +64,74 @@
...
+          max-width: 60px;
+          height: 20px;
+          border-radius: 2px;
+          font-size: 12px;
+          margin-left: 25px;
+          padding: 5px 10px;
+
+          &.Success {
+            background-color: #e1fbff;
+            color: #64c2d1;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss:94]: color template please

File: apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss [hunk 10]
```diff
@@ -62,41 +64,74 @@
...
+          }
+          &.Failure {
+            background-color: #ffcccc;
+            color: #d12221;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/TransComponents/Transactions/Transactions.module.scss:98]: can we add this palettes to the template?

File: apps/web/src/pages/Wallet/TransComponents/Transactions/index.tsx [hunk 11]
```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import styles from './Transactions.module.scss';
-import splitKey from '../../../../helpers/splitKey';
+import splitKey from 'src/helpers/splitKey';
+import classNames from 'classnames';
 
 const data = [
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/TransComponents/Transactions/index.tsx:6]: Can we replace data to out file and import from there?

File: apps/web/src/pages/Wallet/Wallet.module.scss [hunk 12]
```diff
@@ -1,43 +1,48 @@
...
+        margin: 15px 0 20px 0;
+        gap: 12px;
 
         & .Tab {
           text-transform: uppercase;
-          margin: 20px 20px;
-          padding-bottom: 10px;
-          font-size: 18px;
-          font-weight: 700;
+          font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Wallet/Wallet.module.scss:33]: font template

File: apps/web/src/pages/SingleItem/SingleItem.module.scss (hunk 4 above)
Comment (dbjowett) [reply apps/web/src/pages/SingleItem/SingleItem.module.scss:100]: the only difference is font-size, so i cannot use global one

File: apps/web/src/pages/SingleItem/SingleItem.module.scss (hunk 5 above)
Comment (dbjowett) [reply apps/web/src/pages/SingleItem/SingleItem.module.scss:111]: the only difference is font-size, so i cannot use global one

**Review by otabeknos1rov — APPROVED** [review]

**Pull request merged**